components/qr.js

```diff
@@ -16,20 +17,25 @@ export const qrImageSettings = {
 
 export default function Qr ({ asIs, value, useWallet: automated, statusVariant, description, status }) {
   const qrValue = asIs ? value : 'lightning:' + value.toUpperCase()
-  const wallet = useWallet()
+  const { wallets } = useWallets()
 
   useEffect(() => {
     async function effect () {
-      if (automated && wallet) {
-        try {
-          await wallet.sendPayment(value)
-        } catch (e) {
-          console.log(e?.message)
+      const usableWallets = wallets.filter(w => !w.def.isAvailable || w.def.isAvailable())
+        .filter(w => w.config?.enabled && canSend(w))[0]
+      if (automated && usableWallets.length > 0) {
+        for (const wallet of usableWallets) {
+          try {
+            await wallet.sendPayment(value)
+            break
+          } catch (e) {
+            console.log(e?.message)
+          }
```

Comment:
Try every wallet until one successfully pays

Reply:
The logic on use-paid-mutation is there because we are cancelling and recreating the invoices.
In this case we don't do anything like that, so i think we won't need the complex retry logic here, a lightning invoice can be paid only once, right?